nearcore/src/runtime/mod.rs

```diff
@@ -48,9 +48,9 @@ use near_primitives::views::{
     AccessKeyInfoView, CallResult, QueryRequest, QueryResponse, QueryResponseKind, ViewApplyState,
     ViewStateResult,
 };
-use near_store::flat_state::ChainAccessForFlatStorage;
-use near_store::flat_state::{
-    store_helper, FlatStateFactory, FlatStorageCreationStatus, FlatStorageState,
+use near_store::flat::{
+    store_helper, ChainAccessForFlatStorage, FlatStorage, FlatStorageCreationStatus,
```

Comment:
This is one instance where I’d argue that uses of `store_helper` should be further qualified with and used as `flat::store_helper::*` (well, technically all the names therein contain `_flat_` in their name, but it isn’t immediately clear from an use if `store_helper` contains anything more than flat storage helpers…), or alternatively since the names of the helper functions are already well qualified (they contain `flat` in them!) this could be an `use near_store::flat::store_helper::{...}`.)